src/debugger/adapter/DebugAdapter.js

```diff
@@ -126,9 +128,17 @@ class PrepackDebugSession extends DebugSession {
         process.exit();
       },
     };
+
+    this._ensureAdapterChannelCreated("launchRequest");
```

Comment:
This is unnecessary. You just created _adapterChannel before this line, it makes no sense to check it.